src/Dropdown.js

```diff
@@ -194,16 +194,16 @@ class Dropdown extends React.Component {
     }
 
     if (this.props.onToggle) {
-      this.props.onToggle(open);
+      this.props.onToggle(open, eventType);
```

Comment:
i'd make this second argument an object like `{ eventType: 'click', }` that way its more future proof if we want to add mroe stuff there.